lib/constants.go

```diff
@@ -52,4 +52,5 @@ const (
 	RenewPolicyCollection        string = "renewPolicy"            // firestore and bigquery
 	RenewPolicyViewCollection    string = "renewPolicyView"        // bigquery
 	RenewTransactionCollection   string = "renewTransactions"      // firestore and bigquery
+	MailReportCollection         string = "mail-report"            // only for bigquery
```

Comment:
@diogo-wopta here I would renew the table into `mailReport` to set a common naming rule

Reply:
Personally I agree. So I'll go with the renaming since the db will have to be modified anyway.

Reply:
table recreated on bigquery with the new name